app/assets/src/components/views/discovery/DiscoveryHeader.jsx

```diff
@@ -71,30 +71,52 @@ class DiscoveryHeader extends React.Component {
       tabs,
     } = this.props;
 
+    let disabledVisualizations = currentTab === "visualizations";
```

Comment:
Would you mind renaming this variable to something like `disableSidebars` instead? Confused me at first glance 😅

Reply:
Of course! Good suggestion!